build/pages/announcements/announcements.html

```diff
@@ -0,0 +1,158 @@
+                        <p class="sr_only">Search</p>
+                        <img alt="Search" src="/build/assets/icons/magnifier.svg" />
+                    </a>
+                </div>
+            </div>
+        </div>
+    </header>
+    <div class="container container--header_accord">
+        <div class="header_accord" data-js="header_accord">
+            <menu class="link_list link_list--info" data-js="info_menu">
+                <h1 class="link_list__hdng">Information:</h1>
+                <li><a href="/announcements">Announcements</a></li>
+                <li><a href="/events">Events</a></li>
+                <li><a href="/organizations">Organizations</a></li>
+                <li><a href="/usc">About the USC</a></li>
+                <li><a href="/usc/officers">Officers of the USC</a></li>
+            </menu>
+            <menu class="link_list link_list--srvcs" data-js="srvcs_menu">
+                <h1 class="link_list__hdng">Services:</h1>
+                <li><a href="/services/volunteer">Volunteer Opportunities</a></li>
+                <li><a href="/services/education">Education and Training</a></li>
+                <li><a href="/usc/contact">Contact the USC</a></li>
+                <li><a href="/usc-cite/contact">Contact the USC-CITE</a></li>
+            </menu>
+            <form class="search_form" action="/search" method="post" data-js="search_form">
+                <label for="search" class="search_form__label">Search the USC Infosite</label>
+                <input id="search" class="search_form__input" type="text" />
+                <button class="search_form__btn" type="submit">
+                    <img alt="" src="/build/assets/icons/magnifier.svg" /><span class="sr_only">Search the USC Infosite</span>
+                </button>
+            </form>
+        </div>
+    </div>
+
+    <section class="announcements">
+            <h1 class="announcements__heading">Announcements</h1>
+            <p>Including advisories from the various offices of the university.</p>
+        </div>
+    </section>
+
+    <!-- Main section goes here -->
+    <main class="main">
+        <div class="main__container">
+            <div class="main__box">
+                <a class="link" href="#">Free accommodation for students affected on the recent fire incident at Brgy. San Nicolas, La Paz</a>
+                <p class="details office">University Student Council</p>
+                <p class="details">February 20, 2024 <img src="/build/assets/logos/USC-logo.png" alt="" class="office__logo"></p>
+            </div>
+            <div class="main__box">
+                <a class="link" href="#">Civilian clothes are allowed starting 4:00 PM on the onset of Feb Fever 2024</a>
+                <p class="details office">Office of Student Affairs</p>
+                <p class="details">February 13, 2024 <img src="/build/assets/logos/WVSU-Main-Logo.png" alt="" class="office__logo"></p>
```

Comment:
Why not rename `.details` to `.author`? We can encapsulate in in its own `<div>` and assign a class. That way, you can correctly use `office__logo` (which be renamed as such). But what do you think?